src/mscorlib/src/System/Runtime/Intrinsics/X86/Fma.PlatformNotSupported.cs

```diff
@@ -15,140 +15,140 @@ public static class Fma
         public static bool IsSupported { get { return false; } }
 
         /// <summary>
-        /// __m128 _mm_fmadd_ps (__m128 a, __m128 b, __m128 c)
+        /// __m128 _mm_fmadd_ps (__m128 a, __m128 b, __m128 c); VFMADDPS xmm, xmm, xmm/m128
         /// </summary>
         public static Vector128<float> MultiplyAdd(Vector128<float> a, Vector128<float> b, Vector128<float> c) { throw new PlatformNotSupportedException(); }
         /// <summary>
-        /// __m128d _mm_fmadd_pd (__m128d a, __m128d b, __m128d c)
+        /// __m128d _mm_fmadd_pd (__m128d a, __m128d b, __m128d c); VFMADDPD xmm, xmm, xmm/m128
```

Comment:
Putting the instruction in a new line maybe looks better?
```csharp
/// <summary>
/// __m128d _mm_fmadd_pd (__m128d a, __m128d b, __m128d c)
/// VFMADDPD xmm, xmm, xmm/m128
/// </summary>
```

Reply:
Possibly. It doesn't look as good for the intrinsics with multiple mappings, however.

The current format will work fairly well for a search and replace to improve the documentation however.

Reply:
@eerhardt could you suggest the comment format?

Reply:
FWIW, my preference would also be to have the instruction(s) on a separate line.

Reply:
I agree that separate lines are more readable.

Reply:
Fixed